File: CodeConversionCS/CodeConversionCS/Snake.cs
public class Snake : Animal
{
    public bool Poisonous { get; set;}

    public Snake(String name, bool hasFur, int numberOfLegs, double sizeInFeet, bool poisonous) 
        : base(name, hasFur, numberOfLegs, sizeInFeet)
    {
        this.Poisonous = poisonous;
    }

    public override String GetDescription()
    {
        if(Poisonous == true)
        {
            return base.GetDescription() + " Look out! This snake is poisonous!";
        }
        else
        {
            return base.GetDescription();
        }
    }
}

File: code_conversion_js/snake.mjs
import { Animal } from "./animal.mjs";
export { Snake };
class Snake extends Animal
{
    poisonous;
    constructor(name, hasFur, numberOfLegs, sizeInFeet, poisonous)
    {
        super(name, hasFur, numberOfLegs, sizeInFeet);
        this.poisonous = poisonous;
        Object.seal(this);
    }

    getDescription()
    {
        if(this.poisonous)
            return super.getDescription() + " Look out! This snake is poisonous.";
        return super.getDescription();
    }
}